index.html

```diff
@@ -3,16 +3,72 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css"
+    />
+
     <link rel="stylesheet" href="/styles/style.css" />
     <title>ThemeMascot Covid-19 UI</title>
   </head>
-<<<<<<< HEAD
-  <body></body>
-=======
   <body>
+    <!-- Navigation Section-->
     <nav id="nav" class="nav">
-      <div class="container-nav"></div>
+      <div class="container-nav">
+        <div class="brand">
+          <a href="index.html">
+            <img
+              src="./assets/logo-white.png"
+              alt="ThemeMascot"
+              class="brand-img"
+            />
+          </a>
+        </div>
+        <input type="checkbox" class="toggler" id="toggler" />
+        <div class="menu-wrap">
+          <ul class="menu-links">
+            <li><a class="menu-link" href="#home">Home</a></li>
+            <li><a class="menu-link" href="#about">About</a></li>
+            <li><a class="menu-link" href="#symptoms">Symptoms</a></li>
+            <li>
+              <a class="menu-link" href="#prevention">Prevention</a>
+            </li>
+            <li><a class="menu-link" href="#doctors">Doctors</a></li>
+            <li><a class="menu-link" href="#blog">Blog</a></li>
+            <li>
+              <a href="#" class="btn btn-link"
+                ><i class="fa fa-mobile"></i> +347697804</a
+              >
+            </li>
+          </ul>
+        </div>
+        <label for="toggler" class="toggler-label">
+          <span></span>
+        </label>
+      </div>
     </nav>
+    <!--Navigation Section-->
+
+    <!-- Header: Showcase Section-->
+    <header class="showcase">
+      <div class="showcase-container">
+        <div class="showcase-content">
```

Comment:
Since I'm using grid to style the contents of the showcase, showcase-container will be the grid container/parent, while showcase-content has the grid element